packages/dhis2/src/Adaptor.js

```diff
@@ -279,8 +299,8 @@ export function create(resourceType, data, options = {}, callback = false) {
  *   storedBy: 'admin',
  *   dataValues: [],
  * });
- * @example <caption>a trackedEntityInstance</caption>
- * update('trackedEntityInstances', 'IeQfgUtGPq2', {
+ * @example <caption>a tracker entity. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#webapi_nti_import Update tracker docs}</caption>
+ * update('trackedEntityInstances', '', {
```

Comment:
@hunterachieng I'm quite worried about this example :thinking: 
* It should be `update('trackedEntities')`. Although we (should) technically support the old format, we need examples which need the new stuff.
* The actual data structure in the example is outdated. For example, `trackedEntityInstace` (the "id") should now be `trackedEntity`. And `created` should now be `createdAt`. See the [migration guide](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker-deprecated.html#webapi_tracker_migration)
* There is no ID or path in the argument. But we should still need it. The API should not change. This makes the example misleading and wierd.

But also! Looking over the implementation, we don't nest or flatten the input data. So if we POST to `/tracker/` then the server won't understand what we're posting. Also, when we nest data for the new tracker, we need to ensure that the payload includes the id field. We should NOT need the id to be in the payload.

So if you were to run this example through the adaptor on this branch, I'm pretty sure it would fail (for a bunch of reasons!)

We need to take a really close look at all the examples to make sure they're up to date. This is important for users, but also important for us because it should verify that our adaptor works like we think it does!

Reply:
Updated the example